specification/http/1.0/openapi.json

```diff
@@ -0,0 +1,456 @@
+              "description": "The globally-unique ID for this tool execution."
+            },
+            "trace_id": {
+              "type": "string",
+              "description": "A trace identifier for the tool execution."
+            },
+            "tool_id": {
+              "$ref": "#/components/schemas/ToolId"
+            },
+            "input": {
+              "type": "object",
+              "description": "The input parameters for the tool call.",
+              "additionalProperties": true
+            },
+            "context": {
+              "type": "object",
+              "properties": {
+                "authorization": {
+                  "type": "array",
+                  "description": "The authorization information for the tool call.",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "The unique identifier for the authorization method or authorization provider."
+                      },
+                      "token": {
+                        "type": "string",
+                        "description": "The token for the tool call."
+                      }
+                    },
+                    "required": ["id", "token"],
+                    "additionalProperties": false
+                  }
+                },
+                "secrets": {
+                  "type": "array",
+                  "description": "The secrets for the tool call.",
+                  "items": {
+                    "$ref": "#/components/schemas/ToolSecret"
+                  },
+                  "additionalProperties": true
+                },
+                "user_id": {
+                  "type": "string",
+                  "description": "A unique ID that identifies the user, if present."
+                },
+                "user_info": {
+                  "type": "object",
+                  "description": "Additional information about the user, if present."
+                }
+              },
+              "additionalProperties": false
+            }
+          },
+          "required": ["execution_id", "tool_id"],
+          "additionalProperties": false
+        },
+        "ExecuteToolResponse": {
+          "type": "object",
+          "properties": {
+            "$schema": {
+              "type": "string",
+              "format": "uri"
+            },
+            "execution_id": {
+              "type": "string",
+              "description": "The globally-unique ID for this tool execution."
+            },
+            "duration": {
+              "type": "number",
+              "description": "The duration of the tool call, in milliseconds"
+            },
+            "success": {
+              "type": "boolean",
+              "description": "Whether the tool execution was successful."
+            },
+            "output": {
```

Comment:
What do you think about `output` containing `value` and `error` vs. `value` and `error` being in the root level?

Does nesting it under `output` help w/ the idea that the error is an error that was intentionally output by the tool?

Reply:
I like this, I un-nested these in my latest commit.